src/com/namelessmc/java_api/NamelessUser.java

```diff
@@ -86,27 +77,27 @@ public NamelessAPI getApi() {
 	public void invalidateCache() {
 		this.userInfo = null;
 	}
-	
+
 	public int getId() throws NamelessException {
 		if (this.id == null) {
 			this.loadUserInfo();
 			this.id = this.userInfo.get("id").getAsInt();
 		}
-		
+
 		return this.id;
 	}
-	
+
 	public String getUsername() throws NamelessException {
 		if (this.username == null) {
 			this.loadUserInfo();
 			this.username = this.userInfo.get("username").getAsString();
 		}
-		
+
 		return this.username;
 	}
-	
+
 	public Optional<UUID> getUniqueId() throws NamelessException {
-		if (this.uuid == null) {
+		if (this.uuid != null) {
```

Comment:
`==` was correct, if the UUID does *not* exist it should perform an API request to retrieve it. If the UUID is not null it can return it.

Reply:
Yep, I see that this doesn't make sense. Probably made this mistake while reverting the Optional Logic change